test/cppsuite/test_harness/workload/thread_context.h

```diff
@@ -29,115 +29,70 @@
 #ifndef THREAD_CONTEXT_H
 #define THREAD_CONTEXT_H
 
-#include "../core/throttle.h"
+#include <string>
+
+extern "C" {
+#include "test_util.h"
+}
+
+#include "../core/configuration.h"
 #include "../timestamp_manager.h"
-#include "database_model.h"
-#include "random_generator.h"
+#include "../workload/database_model.h"
 #include "workload_tracking.h"
 
+//#include "../core/throttle.h"
```

Comment:
You can remove these.

Reply:
Oops, leftover after fixing compilation issues. Done.